luvit-loader.lua

```diff
@@ -197,7 +197,7 @@ local function loader(dir, path, bundleOnly)
         return package.loaded[key]
       end
       local code = bundle.readfile(fullPath)
-      local module = loadstring(code, key)()
+      local module = loadstring(code, key)(key)
```

Comment:
Looks like I was right--appending '@' to the path results in module not found errors when lit is built (when using the `bundle:` prefix, this command works fine):

```
lit version: 3.4.3
luvi version: v2.7.4
command: ls
fail: C:\Users\Ryan\Programming\Luvit\lit.exe\deps\git\init.lua:2: module './core' not found:
```

Making bundles use the '@' prefix will require some restructuring of how `luvit-loader` checks the caller and decides whether or not to set the `bundleOnly` flag.